### PR TITLE
Optimizations

### PR DESCRIPTION
**Pull request by bfred-it**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -99,31 +99,25 @@ const build = (tagName, attrs, children) => {
...
-		});
+		el.appendChild(children);
 	}
 
 	return el;
 };
 
 function h(tagName, attrs) {
-	attrs = attrs || {};
+	const childrenArgs = [].slice.apply(arguments, [2]);
```
Comment (vadimdemedes) [inline index.js]: Perhaps `[].slice.call(arguments, 2)` would be better?

Comment (bfred-it) [reply index.js]: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#what-is-safe-arguments-usage

Comment (vadimdemedes) [reply index.js]: Ok.

**Comment by vadimdemedes** [comment]
Could you fix up the merge conflict? I'm on a laptop without git yet.

**Pull request merged**